ephypype/commands/neuropycon.py

```diff
@@ -0,0 +1,319 @@
+"""Command line interface for ephypype package"""
+# TODO: autocompletion
+#       consider merging conn and ep2ts nodes
+#       create additional node for plugin option so
+#       the -n option can be invoked only in multiproc
+
+import click
```

Comment:
this should use the `click` from externals folder once we have it

Reply:
I'm creating a PR for it now.

Reply:
Done